pages/studentPortal/labs/multiplication-connect/Index.tsx

```diff
@@ -68,14 +70,21 @@ export const calculateWinner = (
         rows[i - 1][index - 1].selectedBy == player &&
         rows[i - 2][index - 2].selectedBy == player &&
         rows[i - 3][index - 3].selectedBy == player
-          ? console.log(player, "(descending diagonal) Four in a row!")
+          ? (str = `${player} (descending diagonal) Four in a row!`)
```

Comment:
Nice! Your component is easier to test now that it returns a value instead of returning nothing